src/configs/config.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   port: 3000,
-  mongodbUrl: 'mongodb://mongo:27017/follothru',
+  mongodbUrl:
+    'mongodb://follothru:psychology2017@ds217921.mlab.com:17921/follothru',
```

Comment:
don't need to change the mongodbUrl to connect to cloud instance
just change prod to 'true' to connect to cloud db